laravel/app/Http/Kernel.php

```diff
@@ -14,11 +14,13 @@ class Kernel extends HttpKernel
      * @var array
      */
     protected $middleware = [
-        \Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode::class,
+        // \App\Http\Middleware\TrustHosts::class,
```

Comment:
Should this comment be here?

Reply:
Yes. I brought a lot of files up to date with the versions Laravel 8 provides.

See here: https://github.com/laravel/laravel/blob/8.x/app/Http/Kernel.php